packages/apidom-ls/src/services/validation/linter-functions.ts

```diff
@@ -1009,9 +1011,35 @@ export const standardLinterfunctions: FunctionItem[] = [
     functionName: 'apilintOpenAPIPathTemplateValid',
     function: (element: Element) => {
       if (isStringElement(element)) {
-        return true;
+        const pathItemElement = (element.parent as MemberElement).value as ObjectElement;
+
+        if (pathItemElement.length === 0) {
+          return true;
+        }
+
+        const pathTemplateResolveParams: { [key: string]: string } = {};
+
+        pathItemElement.forEach((operation) => {
+          const parameters = (operation as OperationElement).get('parameters');
```

Comment:
This will throw error if `operation` is not `OperationElement` as described above.